mslib/msui/mpl_map.py

```diff
@@ -55,6 +55,8 @@
 OPENAIP_NOTICE = "Airspace data used comes from openAIP.\n" \
                  "Visit openAIP.net and contribute to better aviation data, free for everyone to use and share."
 OURAIRPORTS_NOTICE = "Airports provided by OurAirports."
+mpl_logger = logging.getLogger('matplotlib')
```

Comment:
changing the mpl_logger is the needed solution.

but we need our own debug messages in all mslib modules